### PR TITLE
Service: extract some methods to cmdutils

### PR DESCRIPTION
**Pull request by hsanjuan**
And some small renaming.

### CODE REVIEW

File: cmd/ipfs-cluster-service/main.go [hunk 1]
```diff
@@ -628,23 +628,14 @@ func yesNoPrompt(prompt string) bool {
...
-		cfgHelper.Configs(),
+	cfgHelper, err := cmdutils.NewLoadedConfigHelper(
+		configPath,
+		identityPath,
 	)
-	checkErr("creating state manager", err)
+	checkErr("loading configurations", err)
+	cfgHelper.Manager().Shutdown()
+	mgr, err := cmdutils.NewStateManagerWithHelper(cfgHelper)
+	checkErr("loading state manager", err)
```
Comment (kishansagathiya) [inline cmd/ipfs-cluster-service/main.go]:
```suggestion
	checkErr("creating state manager", err)
```

File: cmdutils/cmdutils.go [hunk 2]
```diff
@@ -56,3 +63,67 @@ func getPort(ln net.Listener, code int) int {
 	}
 	return 0
 }
+
+// HandleSignals orderly shutsdown an IPFS Cluster peer
```
Comment (kishansagathiya) [inline cmdutils/cmdutils.go]:
```suggestion
// HandleSignals orderly shuts down an IPFS Cluster peer
```

File: cmdutils/state.go [hunk 3]
```diff
@@ -26,6 +26,7 @@ type StateManager interface {
 	ImportState(io.Reader) error
 	ExportState(io.Writer) error
 	GetStore() (ds.Datastore, error)
+	GetOfflineState(ds.Datastore) (state.State, error)
```
Comment (kishansagathiya) [inline cmdutils/state.go]: This method is not used anywhere outside this package. Why export it?

Comment (hsanjuan) [reply cmdutils/state.go]: It will be used.

**Pull request merged**